Loop TodayExtension/Base.lproj/MainInterface.storyboard

```diff
@@ -0,0 +1,334 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Great - per our discussion I'm deferring this work and have created #284 to track progress.